### PR TITLE
Basic Flow

### PR DESCRIPTION
**Pull request by casesandberg**

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -1,55 +1,25 @@
...
-} from 'redux';
-import { createLogger } from 'redux-logger'
-import {
-  reducer as asyncDataFetchReducer,
-  middleware as asyncDataFetchMiddleware,
-  storeKey as asyncDataFetchStoreKey,
-  actions as asyncDataFetchActions,
-} from '@hharnisc/async-data-fetch';
+import store from './store'
+import { actions as asyncDataFetchActions } from './store/async-data-fetch-middleware'
```
Comment (hharnisc) [inline frontend/src/index.js]: Why not grab this from the imported package? Looks like a copy :smile:

File: frontend/src/store/async-data-fetch-middleware.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import RPCClient from '@hharnisc/micro-rpc-client';
```
Comment (hharnisc) [inline frontend/src/store/async-data-fetch-middleware.js]: This is a copy of the package right?

Comment (casesandberg) [reply frontend/src/store/async-data-fetch-middleware.js]: It’s not, will explain on our ☎️

File: frontend/src/index.js (hunk 1 above)
Comment (casesandberg) [reply frontend/src/index.js]: i use a transform option on all middlewares that return data now. Wanted to show it to you and get your thoughts on it

**Comment by casesandberg** [comment]
And yes preferably just npm and not fragment with yarn. I hate that CRA does that

**Pull request merged**